docs/getting-started/devenv/get-started-with-wp-now.md

```diff
@@ -1,3 +1,7 @@
+<div class="callout callout-info">
+`wp-now` is still experimental. For a more comprehensive overview of their current status and roadmap, please check <a href="https://github.com/WordPress/wordpress-playground"> WordPress Playground's repository </a> and <a href="https://github.com/WordPress/playground-tools/tree/trunk/packages/wp-now">wp-now's</a>.
```

Comment:
```suggestion
<code>wp-now</code> is still experimental. For a more comprehensive overview of their current status and roadmap, please check <a href="https://github.com/WordPress/wordpress-playground"> WordPress Playground's repository </a> and <a href="https://github.com/WordPress/playground-tools/tree/trunk/packages/wp-now">wp-now's</a>.
```

I think we can not use markdown tags inside the callout section.

Reply:
Yup, markdown cannot be in the callouts.